### PR TITLE
Fixes #33155 - Drop Spice XPI support

### PR DESCRIPTION
**Pull request by ekohl**
This plugin is no longer maintained and relied on Xulrunner support in Firefox. That means that with Firefox 60 or newer it's broken.

### CODE REVIEW

**Comment by upadhyeammit** [comment]
Not having env to try changes however codewise looks good to me!

**Review by ehelms — APPROVED** [review]

**Comment by ekohl** [comment]
Additional context: https://gitlab.freedesktop.org/spice/spice-xpi is archived and the last commit was 6 years ago.

File: app/helpers/compute_resources_vms_helper.rb [hunk 1]
```diff
@@ -44,26 +44,11 @@ def prop(method, title = nil)
...
     }
-    if supports_spice_xpi?
-      options.merge!(
-        :address     => console[:address],
-        :secure_port => console[:secure_port],
-        :subject     => console[:subject],
-        :title       => _("%s - Press Shift-F12 to release the cursor.") % console[:name]
-      )
-    end
-    options[:ca_cert] = URI.escape(console[:ca_cert]) if console[:ca_cert].present?
```
Comment (tbrisker) [inline app/helpers/compute_resources_vms_helper.rb:65]: Looks like this line was missed? Or is it only needed for xpi?

Comment (ekohl) [reply app/helpers/compute_resources_vms_helper.rb:65]: It's only needed for xpi. With NoVNC there is only the regular server certificate.

**Review by tbrisker — APPROVED** [review]
Thanks @ekohl !

**Pull request merged**